RFS/docker/docker-compose.yml

```diff
@@ -8,6 +8,9 @@ services:
     environment:
       - discovery.type=single-node
       - path.repo=/snapshots
+      - AWS_ACCESS_KEY_ID=${access_key}
```

Comment:
is there a reason you're not using the typical env var naming conventions locally? (e.g. not `- AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}` and the corresponding change in the documentation). Not a big deal, but I'm not sure I understand the reason.

Reply:
It's so that you can use a different set of credentials on the host machine if you want to.  I was having trouble with my host machine (mac laptop) picking up and using the creds intended for the RFS Containers, which was pretty annoying because that meant that when I ran the AWS CLI to get renewed creds for the Containers using STS, the CLI would use the expired credential set I was trying to renew! 🙃